src/gui/mainwindow.ui

```diff
@@ -199,7 +199,7 @@
   </action>
   <action name="actionResumeSession">
    <property name="text">
-    <string>&amp;Resume Session</string>
+    <string>R&amp;esume Session</string>
```

Comment:
Just double checking, is this change intended?

Reply:
Designer added ampersands to most (all?) such actions in this file but i reverted them all except for this one and "Shut down". This one conflicts with "Remove" action and "Shut down" conflicts with "Hibernate". With Qt 6.7.2, even in 4.6.7 (for shut down). it seems that these changes are applied automatically in runtime.

Reply:
#15658 is relevant to this I believe